guides/common/modules/proc_configuring-dns-nsupdate.adoc

```diff
@@ -0,0 +1,19 @@
+[id="configuring_dns_nsupdate_{context}"]
+= Configuring dns_nsupdate
+
+The _dns_nsupdate_ DNS provider manages DNS records using nsupdate.
+It can be used with any https://www.rfc-editor.org/rfc/rfc2136[RFC2136] compatible DNS server, but by default ISC BIND is installed.
```

Comment:
```suggestion
You can use _dns_nsupdate_ with any https://www.rfc-editor.org/rfc/rfc2136[RFC2136]-compatible DNS server, but by default ISC BIND is installed.
```
or maybe this would be better to avoid the need to use the hyphen:
```suggestion
You can use _dns_nsupdate_ with any DNS server compatible with https://www.rfc-editor.org/rfc/rfc2136[RFC2136], but by default ISC BIND is installed.
```
You know what, let's split this into two sentences and clarify the second one (I really am not sure who does the installing, I'm going to assume it's dns_nsupdate):
```suggestion
You can use _dns_nsupdate_ with any DNS server compatible with https://www.rfc-editor.org/rfc/rfc2136[RFC2136].
By default, _dns_nsupdate_ installs the ISC BIND server.
```